docs/genai/development/working-with-rag.ipynb

```diff
@@ -1,13 +1,5 @@
 {
```

Comment:
<p>need to remove the cell output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/7594/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>